.gitignore

```diff
@@ -2,7 +2,12 @@
 HydraIRC/libs/libxml2/*
 # Ignore libxml2-old folder, where we save the previous version of libxml
 HydraIRC/libs/libxml2-old/*
+HydraIRC/libs/libxml2-dev/*
```

Comment:
not sure we need this additional entry once the update is applied right?

Reply:
removed